### ISSUE TITLE
[syncfusion_flutter_datagrid] UI Filtering TextStyle

### ISSUE THREAD

**Issue by Alyxsqrd** [syncfusion/flutter-widgets#975]
Please allow us to customize the TextStyles inside the "Sort and Filter" page. This would be useful to change the default text color and font family! 👍

![IMG_3292](https://user-images.githubusercontent.com/25337749/202361934-e4d53351-8bcf-4042-8148-c40a65f54306.jpg)
![IMG_3294](https://user-images.githubusercontent.com/25337749/202361940-d3b75493-4b0a-406b-af85-1394e4ae424c.jpg)

**Comment by Alyxsqrd** [comment]
Looks like the default TextStyles are set here: https://github.com/syncfusion/flutter-widgets/blob/master/packages/syncfusion_flutter_datagrid/lib/src/datagrid_widget/runtime/column.dart#L1837
```dart
class DataGridFilterHelper {
  /// Provides the text style to the tiles.
  TextStyle get textStyle => TextStyle(
      fontSize: 14.0,
      color: textColor,
      fontFamily: 'Roboto',
      fontWeight: FontWeight.normal);

  /// Provides the text style to the disabled tiles.
  TextStyle get disableTextStyle => TextStyle(
      fontSize: 14.0,
      color: disableIconColor,
      fontFamily: 'Roboto',
      fontWeight: FontWeight.normal);
}
```
Is it possible to override this?

**Comment by Alyxsqrd** [comment]
Please expose an SfDataGrid property we can use to change the TextStyles commented above 👍

**Comment by Tamilarasan-Paranthaman** [comment]
Hi @Alyxsqrd,

As of now, DataGrid doesn’t have support for changing the text style in the Filter popup menu. We have considered your request as a feature. This feature will be implemented in our 2023 Volume 1 release, which is expected to be rolled out in April 2023. We will let you know when this feature is implemented. We appreciate your patience until then. You can follow up with the below feedback for further details,

Feedback Link: https://www.syncfusion.com/feedback/39277/support-to-change-the-text-style-and-scroll-the-checked-listbox-content 

Regards,
Tamilarasan

**Comment by Tamilarasan-Paranthaman** [comment]
@Alyxsqrd ,

We are glad to announce that our Essential Studio 2023 Volume 1 Main Release V21.1.0.35 is rolled out. The support for changing the filter popup text style has been included in version 21.1.35.

Pub Link: https://pub.dev/packages/syncfusion_flutter_datagrid/changelog  

**UG Documentation**: https://help.syncfusion.com/flutter/datagrid/filtering#change-the-text-style-of-filter-popup-menu 

**Sample Link**: https://www.syncfusion.com/downloads/support/directtrac/general/ze/sample-612463960 


![image](https://user-images.githubusercontent.com/93904422/227444226-37fc101f-7dab-4251-9b5e-67552f735cf1.png)
